modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,39 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0].
+
+[id="logging-release-notes-5-8-0-enhancements"]
+== Enhancements
+* With this update, you can have finer control over access to workload logs in the LokiStack. (link:https://issues.redhat.com/browse/LOG-3841[LOG-3841])
+
+* With this update, you can configure the alerting rules based on log records by using LogQL.
+(link:https://issues.redhat.com/browse/LOG-3548[LOG-3548])
+
+* With this update, The Loki Operator provides `PodDisruptionBudget` resources for LokiStack that determines the minimum number of pods that must be available per component to ensure normal operations. (link:https://issues.redhat.com/browse/LOG-3839[LOG-3839])
+
+* With this update, you can control the Loki component on cluster nodes to improve performance and resilience by applying a set of default affinity and anti-affinity policies.
```

Comment:
Just wording rephrase
With this update, Reliability of existing LokiStack installations are seamlessly improved by applying a set of default Affinity/Anti-Affinity polices.